.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+---
+name: "TER deployment"
```

Comment:
Please let's synchronize the name of the workflow and the file name. I propose to use "publish" for both so we are close to https://github.com/o-ba/tailor_ext/blob/main/.github/workflows/publish.yml.